0-welcome.ipynb

```diff
@@ -45,7 +45,7 @@
     "\n",
```

Comment:
<p>We should change the logo source to point to <code>"https://raw.githubusercontent.com/dask/dask/main/docs/source/images/dask_horizontal.svg"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jsignell/dask-tutorial-advanced/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>